node/node.go

```diff
@@ -122,6 +129,17 @@ func New(cfg *Config, version string) (*Node, error) {
 		n.services = append(n.services, grpc.NewServer(n.cfg.GRPCPort, n.version, n.db, n.log))
 	}
 
+	if cfg.P2P {
+		privKeyStr, _ := os.LookupEnv("P2P_PRIVATE_KEY")
```

Comment:
Is it possible to pass this in as a flag? Not sure if that's worse from a security point of view.

Reply:
We can add a flag for using keystore files in the future.